pkg/receiver/smartagent/README.md

```diff
@@ -2,50 +2,49 @@
 
 > **Note:** The SignalFx Smart Agent receiver is only supported on x86_64/amd64 platforms.
 
-The Smart Agent Receiver allows you to utilize existing [SignalFx Smart Agent monitors](https://github.com/signalfx/signalfx-agent#monitors)
-as OpenTelemetry Collector metric receivers.  It assumes that you have a properly configured environment with a
-functional [Smart Agent release bundle](https://github.com/signalfx/signalfx-agent/releases/latest) on your system,
-which is already provided for x86_64/amd64 Splunk Distribution of OpenTelemetry Collector
```

Comment:
But the note above says `The SignalFx Smart Agent receiver is only supported on x86_64/amd64 platforms.` Should we mention that again here?

Reply:
sorry, specifically the "installation paths" since the distribution executable doesn't provide them alone.

Reply:
Sorry, I'm a bit confused. Are we trying to address people installing a collector executable on a non-x86/amd64 host, which already had a Smart Agent installed on it? Is this supported?

I'm also confused about what is meant by installation paths, and the linked document doesn't mention them. Can you elaborate?

Perhaps if you could rephrase the statement it would help me understand.

Reply:
yeah, it's ambiguous overall. I'm not sure how accurate "only supported on x86_64/amd64 platforms" is as a whole since the native go monitors can work for other arches but don't think that'd need to be addressed here.

The SA bundle is only deployed by the x86 installers (and made available to the docker images) and the link to the getting started guide is, I believe, the intro to packaging and installation methods that provide the bundle. ARM or other arches users may be surprised by the bundle not being available to them after installing the rpm, for example, and calling that out here helps provide some expectation setting.

Reply:
I see, thanks. I've added some language that hopefully helps folks understand.

Reply:
thanks, this is clearer than the original.